c7n/resources/rds.py

```diff
@@ -832,6 +832,51 @@ def set_retention_window(self, resource, retention, copy_tags):
             CopyTagsToSnapshot=copy_tags)
 
 
+@actions.register('set-publicly-accessible')
+class RDSSetPublicAvailability(BaseAction):
+    """
+    This action allows for toggling an RDS instance
+    'PubliclyAccessible' flag to true or false
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: enable-rds-public-accessibility
+                resource: rds
+                filters:
+                  - PubliclyAccessible: false
+                actions:
+                  - type: set-publicly-accessible
+                    accessbile: true
+    """
+
+    schema = type_schema(
+        "set-publicly-accessible",
+        accessible={'type': 'boolean'})
```

Comment:
for most of the boolean, we've been doing `state` instead of idiosyncratic flags per actions, ie state: true/false